x/sp/types/codec.go

```diff
@@ -14,6 +14,7 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgEditStorageProvider{}, "sp/EditStorageProvider", nil)
 	cdc.RegisterConcrete(&MsgUpdateSpStoragePrice{}, "sp/UpdateSpStoragePrice", nil)
 	cdc.RegisterConcrete(&DepositAuthorization{}, "sp/DepositAuthorization", nil)
+	cdc.RegisterConcrete(&MsgUpdateParams{}, "sp/MsgUpdateParams", nil)
```

Comment:
Why is only this one with the `Msg` prefix? And isn't Amino already deprecated?

Reply:
Can we just remove this func `func RegisterCodec(cdc *codec.LegacyAmino)` ?